src/patternfly/components/Wizard/wizard-title.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="pf-c-wizard__title{{#if wizard-title--modifier}} {{wizard-title--modifier}}{{/if}}"
+  {{#if wizard-title--attribute}}
+    {{{wizard-title--attribute}}}
+  {{/if}}>
+  {{#if wizard-title-text--value}}
+    {{#> wizard-title-text}}
+      {{{wizard-title-text--value}}}
+    {{/wizard-title-text}}
+  {{/if}}
```

Comment:
It looks like if there was a partial block and `wizard-title-text--value` we'd get both - do we want that?

Reply:
Good catch; I can't imagine we would. Just pushed an update to resolve that.